server/@types/express/index.d.ts

```diff
@@ -19,10 +20,6 @@ declare module 'express-session' {
     returnTo: string
     nowInMinutes: number
     requestLinkForm: RequestLinkForm
-    slmToken: string
-    validSlmToken: boolean
-    barcodeUserEmail: string
-    barcodeUserOrganisation: string
```

Comment:
These have been refactored into a `barcodeUser` object. The bulk of the PR relates to that refactoring.